sigh/lib/sigh/options.rb

```diff
@@ -114,6 +114,16 @@ def self.available_options
                                      env_name: "SIGH_PROVISIONING_PROFILE_NAME",
                                      description: "The name of the profile that is used on the Apple Developer Portal",
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :keychain_path,
+                                     short_option: "-k",
+                                     env_name: "SIGH_KEYCHAIN_PATH",
+                                     description: "Path to a custom keychain",
```

Comment:
Can this be a bit more specific, perhaps to indicate when the custom keychain would be used, or what for? It may be clear to advanced users, but maybe not the average user, not sure 😅